README.md

```diff
@@ -88,3 +88,44 @@ Visit our documentation on [readthedocs.org](http://ralph-ng.readthedocs.org)
 ## Getting help
 
 Ralph community will answer your questions on a chat: [![Gitter](https://img.shields.io/gitter/room/gitterHQ/gitter.svg)](https://gitter.im/allegro/ralph?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+
+# Dev Tips & Tricks
+
+## how to see asset boxes in front dashboard
+
```

Comment:
One paragraph of intro required - for example: 

By default, when you visit front front dashboard, ...

To overcome this problem, ...